internal/services/sentinel/azuresdkhacks/threatintelligenceindicator.go

```diff
@@ -29,7 +29,7 @@ func (client ThreatIntelligenceIndicatorClient) Get(ctx context.Context, resourc
 			Constraints: []validation.Constraint{{Target: "workspaceName", Name: validation.MaxLength, Rule: 90, Chain: nil},
 				{Target: "workspaceName", Name: validation.MinLength, Rule: 1, Chain: nil},
 				{Target: "workspaceName", Name: validation.Pattern, Rule: `^[A-Za-z0-9][A-Za-z0-9-]+[A-Za-z0-9]$`, Chain: nil}}}}); err != nil {
-		return result, validation.NewError("securityinsight.ThreatIntelligenceIndicatorClient", "Get", err.Error())
+		return result, validation.NewError("securityinsight.ThreatIntelligenceIndicatorClient", "Get", err.Error()) // nolint: govet
```

Comment:
I'm just wondering what `govet` is flagging on these?

Reply:
```
internal/services/sentinel/azuresdkhacks/threatintelligenceindicator.go:32:98: printf: non-constant format string in call to github.com/Azure/go-autorest/autorest/validation.NewError (govet)
                return result, validation.NewError("securityinsight.ThreatIntelligenceIndicatorClient", "Get", err.Error())
```
basically incase it returns a format string within the format string creating a security issue, which isn't an issue here

Reply:
Understood!